.gitignore

```diff
@@ -15,3 +15,132 @@ project/plugins/project/
 # Scala-IDE specific
 .scala_dependencies
 .worksheet
+### JetBrains template
```

Comment:
Just for information there is common gitignore definition: [Scala gitignore](https://github.com/github/gitignore/blob/master/Scala.gitignore)